tests/sentry/api/endpoints/test_user_authenticator_enroll.py

```diff
@@ -268,7 +268,7 @@ def test_u2f_disallow_new_enrollment(self):
         )
 
 
-@control_silo_test(stable=True)
+@control_silo_test(stable=True, include_monolith_run=True)
```

Comment:
:clap: This is a nice way to retain coverage for the small number of monolith paths that we have.